.github/workflows/Downgrade.yml

```
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        version: ['1']
+        version: ['1.10']
```

```suggestion
        version: ['min']
```
would maybe be reasonable for a downgrade action?